src/transaction/log_applier.c

```diff
@@ -6177,6 +6177,8 @@ la_log_record_process (LOG_RECORD_HEADER * lrec, LOG_LSA * final, LOG_PAGE * pg_
       break;
 
     case LOG_DUMMY_CRASH_RECOVERY:
+    case LOG_START_ATOMIC_REPL:
+    case LOG_END_ATOMIC_REPL:
```

Comment:
In this case, let the switch fall to default and ignore the log record. The notification used here for crash log record is not needed.
```suggestion
```